mozdownload/scraper.py

```diff
@@ -49,7 +63,10 @@
                            'android-api-11': 'apk',
                            'android-api-15': 'apk',
                            'android-api-16': 'apk',
```

Comment:
All those older APIs were only needed for Fennec and can be removed. The earliest [Fenix beta is 97.0.0](https://archive.mozilla.org/pub/fenix/releases/97.0.0-beta.1/android/) and we should support all its flavors only.